infra/recipes/terraform/azure/modules/helm_release/main.tf

```diff
@@ -0,0 +1,55 @@
+resource "helm_release" "tables" {
+  chart = "tables"
+  name = "openhouse-tables-service"
+  repository = "../../../../k8s/helm"
+
+  values = ["${file("../../../../k8s/environments/azure/sandbox/tables/values.yaml")}"]
```

Comment:
Looks like we have common prefix. Should be good to extract at root path for readability.

Reply:
Yes I added a local variable